packages/components/src/DataList/DataList.stories.tsx

```diff
@@ -21,7 +21,7 @@ export default {
     viewMode: "story",
   },
   // Comment this out to make it show up in storybook
-  excludeStories: ["Basic", "EmptyState"],
+  // excludeStories: ["Basic", "EmptyState"],
```

Comment:
Don't forget to uncomment this out

```suggestion
  excludeStories: ["Basic", "EmptyState"],
```